src/modules/dpos/validator/components/LatestVotes/index.test.js

```diff
@@ -1,60 +1,69 @@
 import { mountWithRouter } from 'src/utils/testHelpers';
-import { delegateList } from '@tests/constants/delegates';
 import { mockTransactions } from 'src/modules/transaction/__fixtures__';
+import { useTransactions } from 'src/modules/transaction/hooks/queries';
+import { useDelegates } from '../../hooks/queries';
 import LatestVotes from './index';
+import { mockDelegates } from '../../__fixtures__';
 
-const delegateDataList = delegateList(10);
-const delegateData = Object.assign({}, ...delegateDataList);
-
-const props = {
-  votes: {
-    isLoading: false,
-    data: mockTransactions.data.slice(0, 10),
-    loadData: jest.fn(),
-    clearData: jest.fn(),
-    urlSearchParams: {},
-    meta: { total: delegateList.length, count: 10, offset: 0 },
-  },
-  t: str => str,
-  delegates: { data: delegateData },
-};
+jest.mock('src/modules/transaction/hooks/queries');
+jest.mock('../../hooks/queries');
 
 describe('Latest votes', () => {
+  const mockFetchNextDelegates = jest.fn();
+  const mockFetchNextTransactions = jest.fn();
+  const props = {
+    filters: {},
+  };
+
   it('displays initial table of votes', () => {
+    useDelegates.mockReturnValue({
+      data: mockDelegates,
+      isFetching: false,
+      fetchNextPage: mockFetchNextDelegates,
+      hasNextPage: false,
+    });
+    useTransactions.mockReturnValue({
+      data: { ...mockTransactions, data: mockTransactions.data.slice(0, 10) },
+      isFetching: false,
+      fetchNextPage: mockFetchNextTransactions,
+      hasNextPage: false,
+    });
```

Comment:
Please don't mock query hooks follow the discussions below
https://github.com/mswjs/msw/discussions/783